### PR TITLE
set the displayName and __ui__ of the Link component

### LINKED ISSUE #2877: `displayName` is not set for the component of "@yamada-ui/link"
script: component-display-name
package: @yamada-ui/link

### Description
The component "@yamada-ui/link" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

### PR DESCRIPTION
**Pull request by kurakke**
Closes #2877

### CODE REVIEW

**Review by illionillion — DISMISSED** [review]

File: .changeset/flat-actors-wonder.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/link": patch
+---
+
+Set the DisplayName and **ui** of the Link component
```
Comment (hirotomoyamada) [inline .changeset/flat-actors-wonder.md:5]: For unique terms, wrap the word with "`". For example, "`__ui__`".

File: .changeset/flat-actors-wonder.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/link": patch
```
Comment (hirotomoyamada) [inline .changeset/flat-actors-wonder.md:2]: It is "@yamada-ui/next", not `"@yamada-ui/link"`.

File: .changeset/flat-actors-wonder.md (hunk 1 above)
Comment (kurakke) [reply .changeset/flat-actors-wonder.md]: fixed
[0b5592f](https://github.com/yamada-ui/yamada-ui/pull/3034/commits/0b5592f5037d778a1a608a9c991acdfaa1f7bf41)

File: .changeset/flat-actors-wonder.md (hunk 2 above)
Comment (kurakke) [reply .changeset/flat-actors-wonder.md]: fixed
[0b5592f](https://github.com/yamada-ui/yamada-ui/pull/3034/commits/0b5592f5037d778a1a608a9c991acdfaa1f7bf41)

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**